doc/main/overview.rst

```diff
@@ -170,6 +172,16 @@ with the single Coulomb scattering model, is an implementation of the mixed
 simulation algorithm. It is the default model in Geant4 above 100 MeV and
 currently under development in Celeritas.
 
+Optical Physics
+===============
+
+TODO:
```

Comment:
This is a TODO for later and not something you meant to add in this PR, correct?

Reply:
Yeah, just something we need to remember to do — I think even now some of the basic ordering descriptions would be helpful but maybe we should fill this in once we have the initial optical loop in place.